server/src/api/read-ngnews.ts

```diff
@@ -1,12 +1,27 @@
+import { readSchematicSchema } from './read-schematic-collections';
+import { join } from 'path';
+
+console.log(
+  join(__dirname, '../../assets/@nrwl-schematics-ng-new-schema.json')
+);
+
 export function schematicCollectionsForNgNew() {
   return [
     {
       name: '@schematics/angular',
-      description: 'Default Angular CLI workspace'
+      description: 'Default Angular CLI workspace',
+      schema: readSchematicSchema(
+        '@schematics-angular-ng-new-schema.json',
+        join(__dirname, '../../assets/')
+      )
     },
     {
       name: '@nrwl/schematics',
-      description: 'Enterprise-ready Angular CLI workspace'
+      description: 'Enterprise-ready Angular CLI workspace',
+      schema: readSchematicSchema(
+        '@nrwl-schematics-ng-new-schema.json',
```

Comment:
I don't see `@nrwl-schematics-ng-new-schema.json`. Is it supposed to come with this PR as well?

Reply:
Removed